doc-content/enterprise-only/admin-and-config/managing-business-central-database-connection-con.adoc

```diff
@@ -0,0 +1,23 @@
+[id='managing-business-central-database-connection-con']
+= Database connection
+
+{CENTRAL} provides a dedicated `UserGroupCallback` implementation for database server with {PRODUCT} to enable the user task service to retrieve information on users and groups (roles) directly from databases.
+
+You can configure the following database `UserGroupCallback` implementation properties:
+
+[caption="Table 1. "]
+.Database UserGroupCallback properties
+[%header,cols=2]
+[%autowidth]
+|===
+|Property |Description
+
+|`*db.ds.jndi.name*` |JNDI name of the data source used for connections.
+
+|`*db.user.query*` |This property is used to verify the user existence.
```

Comment:
Suggested rewrite:
Verifies the user existence
 
//suggest changing to short sentences as we have in our document. Also, it should not be ending with a period.